agent/acs/handler/acs_handler_test.go

```diff
@@ -519,8 +519,10 @@ func TestHandlerReconnectDelayForInactiveInstanceError(t *testing.T) {
 		mockWsClient.EXPECT().Connect().Do(func() {
 			reconnectDelay := time.Now().Sub(firstConnectionAttemptTime)
 			t.Logf("Delay between successive connections: %v", reconnectDelay)
-			if reconnectDelay < inactiveInstanceReconnectDelay {
-				t.Errorf("Reconnect delay %v is not less than inactive instance reconnect delay %v", reconnectDelay, inactiveInstanceReconnectDelay)
+			// On window platform, we found issue with time.Now().Sub(...) reporting 199.9989 even after the code has already waited for time.NewTimer(200)ms.
+			timeSubFuncSlopAllowed := 1 * time.Millisecond
```

Comment:
I'm not a fan of doing this math here. Can you please use `assert.WithinDuration` instead? https://godoc.org/github.com/stretchr/testify/assert#Assertions.WithinDuration

Writing less code is always better :)